examples/create-react-app/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "graphql-hooks": "3.3.0",
+    "graphql-hooks-memcache": "1.1.0",
+    "prop-types": "15.7.2",
+    "react": "16.8.4",
+    "react-dom": "16.8.4",
+    "react-scripts": "2.1.8"
+  },
+  "scripts": {
+    "start": "react-scripts start",
```

```suggestion
    "start": "SKIP_PREFLIGHT_CHECK=true react-scripts start",
```

Making this change is the only way I can get this to run.